.env.sample

```
@@ -273,11 +273,13 @@ CONSUMER_CONFIG_MAX_DELIVER=4
 AGGREGATE_STREAM=aggregate
 DID_STREAM=did-notify
 PULL_CONSUMER=hub-pull-consumer
+STATUS_LIST_HOST=
 
 NATS_CREDS_FILE=/platform/app_user.creds
 # 'nkey' | 'creds' | 'usernamePassword' | 'none'
 NATS_AUTH_TYPE=nkey
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NATS_AUTH_TYPE key should go before the NATS_CREDS_FILE key

(UnorderedKey)